.golangci.yml

```diff
@@ -0,0 +1,49 @@
+service:
+  golangci-lint-version: 1.21.0 # Keep this in sync with .circleci/config.yml and Jenkinsfile.pr
+
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  gofmt:
+    simplify: true
+  govet:
+    check-shadowing: true
+    enable-all: true
+
+linters:
+  disable-all: true
+  enable:
+    # - bodyclose
+    - deadcode
+    - depguard
```

Comment:
You didn't configure a list for `depguard`. Without a configured list, the linter will do nothing. Did you intent to blacklist any dependencies?

Reply:
Ah, good catch. I literally just enabled everything and fixed what I could. I didn't realize depguard required further configuration. Will remove.